app.test.js

```diff
@@ -63,3 +63,39 @@ describe('GET /api/topics', () => {
 });
 
 
+describe('GET /api', () => {
+  it('should respond with status 200 and list all available endpoints', () => {
+    return request(app)
+      .get('/api')
+      .then((response) => {
+        expect(response.status).toBe(200);
+        
+        // Get the actual routes defined in app
+        const appRoutes = Object.keys(app._router.stack)
+          .filter(route => route.route)
+          .map(route => route.route.path);
+
+        // Extract endpoints from the expected JSON object
+        const expectedEndpoints = Object.keys(endpointsJSON).map(endpoint => endpoint.split(' ')[1]);
+
+        // Compare the actual routes with expected endpoints
+        const endPoints = expectedEndpoints.filter(endpoint => !appRoutes.includes(endpoint));
+
+        expect(endPoints).toEqual(["/api", "/api/topics", "/api/articles"]);
+      });
+  });
```

Comment:
You are already referencing the existing file contents in the test below by requiring them and comparing the object. We also want this to be dynamic which importing from the endpoints.json will accomplish. We should not need the above test because we have the below test.